crates/holochain_sqlite/src/sql/peer_meta_store/get.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  meta_value
+FROM
+  peer_meta
+WHERE
+  peer_url = ?
+  AND meta_key = ?
+  AND (
+    expires_at IS NULL
+    or expires_at >= unixepoch()
```

Comment:
I'm totally fine with how you've done this. We don't need microsecond precision for the expiry. But in case we wanted to make this consistent with the other holochain database code, we picked i64 micros because they fit in the sqlite INTEGER type. So we could instead not divide the micros into seconds then either multiply the unixepoch() or pass in the `now` as a parameter.

Reply:
Updated to work with micros